processor/processor.go

```diff
@@ -214,7 +214,7 @@ func (proc *HandleT) newUserTransformationStat(sourceID, workspaceID string, des
 	numEvents := proc.statsFactory.NewTaggedStat("proc_num_ut_input_events", stats.CountType, tags)
 	numOutputSuccessEvents := proc.statsFactory.NewTaggedStat("proc_num_ut_output_success_events", stats.CountType, tags)
```

Comment:
we still have 
* `proc_num_ut_input_events` & `proc_num_dt_input_events`
* `proc_num_ut_output_success_events` & `proc_num_dt_output_success_events`
* `proc_num_ut_output_failed_events` & `proc_num_dt_output_failed_events`

Is it hard to consolidate these as well?